x-pack/legacy/plugins/code/server/plugin.ts

```diff
@@ -70,19 +71,18 @@ export class CodePlugin {
   private codeServices: CodeServices | null = null;
   private nodeService: NodeRepositoriesService | null = null;
 
-  constructor(initializerContext: PluginInitializerContext) {
+  constructor(private readonly initContext: PluginInitializerContext<PluginSetupContract>) {
     this.log = {} as Logger;
     this.serverOptions = {} as ServerOptions;
   }
 
-  // TODO: options is not a valid param for the setup() api
-  // of the new platform. Will need to pass through the configs
-  // correctly in the new platform.
-  public setup(core: CoreSetup, options: any) {
+  public setup(core: CoreSetup) {
     const { server } = core.http as any;
 
-    this.log = new Logger(server);
-    this.serverOptions = new ServerOptions(options, server.config());
+    // @ts-ignore
```

Comment:
I believe this fixes all the problem with typings in this file:
```diff
- constructor(private readonly initContext: PluginInitializerContext<PluginSetupContract>) {}
+ constructor(private readonly initContext: PluginSetupContract) {}
```
you don't use new platform context here https://github.com/restrry/kibana/blob/109192202c633f180209e5fed0bae600a0d14c32/x-pack/legacy/plugins/code/index.ts#L68

Reply:
done